src/main/java/core/basesyntax/FileWork.java

```
@@ -9,6 +16,20 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        String[] arr = new String[0];
+        try {
+            arr = Files.readString(Path.of(fileName)).toLowerCase().split("\\W+");
+        } catch (IOException e) {
+            e.printStackTrace();
```

don't ignore exception you should re-throw it